src/components/Checkbox/tests/Checkbox.test.tsx

```diff
@@ -52,57 +39,25 @@ describe('<Checkbox />', () => {
       );
 
       (element.find('input')!.domNode as HTMLInputElement).checked = true;
-      element.find(Choice)?.trigger('onClick');
-
-      expect(spy).toHaveBeenCalledWith(false, 'MyCheckbox');
-    });
-
-    it('is called when space is pressed', () => {
-      const spy = jest.fn();
-      const element = mountWithApp(
-        <Checkbox id="MyCheckbox" label="Checkbox" onChange={spy} />,
-      );
-
-      element.find('input')!.trigger('onKeyUp', {
-        keyCode: Key.Space,
-      });
-
-      expect(spy).toHaveBeenCalledTimes(1);
-    });
-
-    it('is not from keys other than space', () => {
-      const spy = jest.fn();
-      const element = mountWithApp(
-        <Checkbox id="MyCheckbox" label="Checkbox" onChange={spy} />,
-      );
-
-      element.find('input')!.trigger('onKeyUp', {
-        keyCode: Key.Enter,
+      const event = new MouseEvent('click', {
+        view: window,
+        bubbles: true,
+        cancelable: true,
       });
+      element.find('input')!.domNode?.dispatchEvent(event);
 
-      expect(spy).not.toHaveBeenCalled();
+      expect(spy).toHaveBeenCalledWith(false, 'MyCheckbox');
     });
 
     it('sets focus on the input when checkbox is toggled off', () => {
       const checkbox = mountWithApp(
         <Checkbox checked id="checkboxId" label="Checkbox" onChange={noop} />,
       );
-      checkbox.find(Choice)!.trigger('onClick');
+      checkbox.find('input')!.trigger('onClick');
 
       expect(document.activeElement).toBe(checkbox.find('input')!.domNode);
     });
 
-    it('is not called from keyboard events when disabled', () => {
-      const spy = jest.fn();
-      const checkbox = mountWithApp(
-        <Checkbox label="label" disabled onChange={spy} />,
-      );
-      checkbox.find('input')!.trigger('onKeyUp', {
-        keyCode: Key.Enter,
-      });
-      expect(spy).not.toHaveBeenCalled();
-    });
```

Comment:
Since we no longer manually handler the keyUp events, we no longer need to test this.